src/components/Alert/Alert.jsx

```diff
@@ -88,9 +88,9 @@ Alert.propTypes = {
   /** [Developer] Adds data-testId attributes for component testing */
   testMode: PropTypes.bool,
   /** An extremely brief title for the alert */
-  title: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
```

Comment:
There is a test that `title` is not required, but it was still required. This fixes that.